_content/gateways/registration.md

```diff
@@ -4,47 +4,49 @@ title: Gateway Registration
 
 # Gateway Registration
 
-We are transitioning to a new way to register and manage your gateways. Please read this guide carefully to understand how it currently works and what will change.
+The are mainly two types of packet forwarders that can be running on your gateway, namely [TTN Packet Forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/master) (formerly known as the Gateway Connector) or [UDP Packet Forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/legacy) (also known as the Semtech Packet Forwarder or Poly Packet Forwarder). 
 
-> At the moment, most gateways don't support the Gateway Connector protocol yet. Follow the steps for connecting using the [Via Packet Forwarder](#via-packet-forwarder).
+Please read this guide carefully to understand how to register your gateway with The Things Network.
 
-## Via Gateway Connector
-This is a work in process. The Things Gateway will be the first gateway to use this new protocol. Firmware and guides for other gateways will need to be updated.
+> If you have an off-the-shelf gateway with the default software, it most likely uses the UDP Packet Forwarder. Follow the steps for connecting using the [UDP Packet Forwarder](#via-udp-packet-forwarder).
 
-> Make sure your gateway supports [The Things Network Gateway Connector](https://github.com/TheThingsNetwork/ttn-gateway-connector) before you continue.
+## Via TTN Packet Forwarder
+The Things Gateway uses this packet forwarder and we also have support for Kerlink, Multitech and other Linux based gateways. This is the preferred packet forwarder to use with The Things Network. Firmware and guides for other gateways will need to be updated.
+
+> Make sure your gateway runs [The Things Network Packet Forwarder](https://github.com/TheThingsNetwork/packet_forwarder/tree/master) before you continue.
```

Comment:
In fact, it's the other way around. You need to register the gateway first before configuring and running it